src/Microsoft.Framework.DependencyInjection.Abstractions/project.json

```diff
@@ -8,7 +8,7 @@
         "net45": {
         },
         "dnx451": { },
```

Comment:
We're still waiting to see how far back dotnet should go. Today it's net46 but it might be net45